lib/apiservers/portlayer/swagger.json

```diff
@@ -1931,7 +1933,7 @@
 		"/tasks/binding": {
 			"post": {
 				"description": "Activate a task that exists within the context of the provided handle",
-				"summary": "Activate an existing task",
+				"summary": "Activate a existing task",
```

Comment:
This is supposed to be `an existing task` - https://www.quickanddirtytips.com/education/grammar/a-versus-an

Reply:
yeah that was an accidental change. Thanks for the link haha